web-client/app/styles/main.scss

```diff
@@ -1153,6 +1153,7 @@ ul.cards-list-container {
         margin: 10px;
         display: block;
         padding-top: 1em;
+        overflow: scroll;
```

Comment:
That's weird, cause Chrome on my Mac is working fine. Make sure you open the page in incongnito _so you don't have any extensions running, that might be conflicting with some JS on the website_